### DISCUSSION TITLE
kpanda/user-guide/gpu/volcano/volcano_binpack/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#6089] [Announcements]
# kpanda/user-guide/gpu/volcano/volcano_binpack/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/gpu/volcano/volcano_binpack/

**Comment by qijuan316** [comment]
按照公式算出来，N1>N2。公式是否正确？
